capstone/src/main/webapp/js/calendar.js

```diff
@@ -0,0 +1,78 @@
+const script = document.createElement('script');
+script.src = 'https://www.gstatic.com/charts/loader.js';
+script.defer = true;
+
+window.initMap = function() {
+  google.charts.load('current', {'packages': ['calendar', 'timeline']});
+  google.charts.setOnLoadCallback(drawCharts);
+
+  /**
+   * Callback function to generate charts.
+   */
+  function drawCharts() {
+    const freqData = new google.visualization.DataTable();
+    const timelineData = new google.visualization.DataTable();
+    fetch('/suggested-cal').
+        then((response) => response.json()).
+        then((deadlineInfo) =>
+          populateDataTables(freqData, timelineData, deadlineInfo));
+    drawFrequencyChart(freqData);
+    drawTimelineChart(timelineData);
+  }
+
+  /**
+   * Generate frequency calendar chart of deadlines.
+   * @param {google.visualization.DataTable} freqData
```

Comment:
Check the suggested line wrapping:
https://google.github.io/styleguide/jsguide.html#jsdoc-line-wrapping
Same comment for other instances of @param and @return